instance-segmentation/requirements.txt

```diff
@@ -0,0 +1,4 @@
+cython==0.29.30
+pycocotools==2.0.4
+torchvision
+segments-ai==0.70
```

Comment:
Do we need Layer here?

Reply:
It breaks the remote build because of Layer